src/include/btmem.h

```diff
@@ -207,8 +208,8 @@ struct __wt_ovfl_txnc {
  * the row-store key is relatively large.
  */
 #define	WT_LAS_FORMAT							\
-    "key_format=" WT_UNCHECKED_STRING(IuQQu)				\
-    ",value_format=" WT_UNCHECKED_STRING(QBu)
+    "key_format=" WT_UNCHECKED_STRING(IuQQuu)				\
+    ",value_format=" WT_UNCHECKED_STRING(QuBu)
```

Comment:
No, the change to the lookaside table is unconditional (it always contains a timestamp field), but the item is always empty when timestamps are disabled.